components/Notice/styled.tsx

```diff
@@ -0,0 +1,88 @@
+import { Block, Button, Typography } from 'prensa'
+import React from 'react'
+
+import {
+  ContainerProps,
+  NoticeProps,
+  
+} from './types'
+
+export const ButtonPdf = ({ children, ...otherProps  }: NoticeProps) => (
+  <Button
+    size='40px'
+    lg={{ size: '32px' }}
+    {...otherProps}
+  >
+    {children}
+  </Button>
+)
+
+export const ButtonWrap = ({ children }: NoticeProps) => (
+  <Block
+    align='column'
+    alignx='left'
+    width='100%'
+    lg={{ custom: 'align-items: flex-start', width: 'auto' }}
+  >
+    {children}
+  </Block>
+)
+
+export const Container = ({ children, ...otherProps }: ContainerProps) => (
+  <Block
+    align='column'
+    aligny='middle'
+    alignx='center'
+    px={2}
+    py={2}
+    custom='gap: 16px'
```

Comment:
Faltou o ponto e vírgula por ser CSS
```suggestion
    custom='gap: 16px;'
```